homework/src/index.js

```diff
@@ -1,47 +1,119 @@
+          class: "forks"
+        });
+        crnap("div", sectLinfobox, {
+          html:
+            null == changeEvent.updated_at
+              ? "Update is unknown"
+              : `Updated: ${new Date(changeEvent.updated_at).toLocaleString()}`,
+          class: "updated"
+        });
+        sectLinfobox = crnap("div", sectLinfobox, {
+          html: null == changeEvent.size ? "No size" : `Size: ${changeEvent.size}`,
+          class: "size"
+        });
+        fetchJSON(changeEvent.contributors_url, (err, contributors) => {
+          let fetchToDo = contributors.length;
+          contributors.forEach(contributor => {
+            fetchJSON(contributor.url, (err, user) => {
+              Object.assign(contributor, user);
+              --fetchToDo;
+              0 === fetchToDo &&
+                (console.log(contributors),
+                  contributors.forEach(contrUser => {
+                    let sectRcontrs = crnap("div", sectR, { id: "contributors" });
+                    sectRcontrs = crnap("div", sectRcontrs, { class: "card" });
+                    let secRimglink = crnap("a", sectRcontrs, { href: contrUser.html_url });
+                    secRimglink = crnap("figure", secRimglink, {});
+                    crnap("img", secRimglink, { src: contrUser.avatar_url });
+                    crnap("h2", sectRcontrs, { html: null == contrUser.name ? "" : `name: ${contrUser.name}` });
+                    crnap("p", sectRcontrs, { html: null == contrUser.login ? "" : `nickname: ${contrUser.login}` });
+                    crnap("p", sectRcontrs, {
+                      html: null == contrUser.location ? "" : `location: ${contrUser.location}`
+                    });
+                    crnap("p", sectRcontrs, {
+                      html: null == contrUser.followers ? "" : `followers: ${contrUser.followers}`
+                    });
+                    crnap("p", sectRcontrs, {
+                      html: null == contrUser.company ? "" : `company: ${contrUser.company}`
+                    });
+                    crnap("p", sectRcontrs, {
+                      html: null == contrUser.contributions ? "" : `contributions: ${contrUser.contributions}`
+                    });
+                    crnap("p", sectRcontrs, {
+                      html:
+                        null == contrUser.updated_at
+                          ? ""
+                          : `last github visit: ${new Date(contrUser.updated_at).toLocaleString()}`
+                    });
+                  }));
+            });
+          });
+        });
       }
-    };
-    xhr.onerror = () => cb(new Error('Network request failed'));
-    xhr.send();
+      const root = document.getElementById("root");
+      let header = crnap("header", root, {});
+      header = crnap("div", header, { class: "selectfield" });
+      const menu = crnap("select", header, { id: "selectmenu" });
+      var main = crnap("main", root, { id: "main" });
```

Comment:
Don't reuse the name `main`. You have already used it for the function `main()`.

Also, don't use `var`. Use `let` and `const` instead.